src/data/data.hpp

```diff
@@ -85,33 +84,38 @@ struct TemperatureData : public Sensor {
 struct Sensors : public Module {
   static constexpr int kNumImus = 4;
   static constexpr int kNumKeyence = 2;
+  static constexpr int kFIFO;
```

Comment:
What is this for? You probably need to assign it some value.

Reply:
Gregory said to leave an unassigned value for now, as he will assign one later. Ask him what it's for, he made the request to have it in the data structure. It's for the vector that gets initialized afterwards, though I'm not sure what the vector is actually for.

Reply:
@gregorydayao12 , Can you come up with a more descriptive name for the constexpr, or add an comment?

@DanGrizzly , please, make sure everything compiles before committing/pushing.

Reply:
I can't do anything about sensors code.

Reply:
If you change code in `data` that is used by sensors, somebody needs to update the sensors' code as well. We cannot approve PRs that do not compile.

If you cannot change the sensors code you need to collaborate with some sensors people to make the PR/feature branch compile.

Reply:
@gregorydayao12 @DanGrizzly 

Wouldn't a better name be `kFifoSize` or something like that?

In general, when your changes break someone else's code, there are several things you can do:
1. Ideally you agree with the other party and they make the necessary changes.
2. You agree with the other party and make changes yourself.
3. You go ahead and make some necessary changes yourself. In this case you need to make extra sure that the other party reviews it and they are happy with the changes to their code before merging.

In general, you can modify other people's code. We might need to move people around later anyway so it's good to get familiar with other modules.